CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## 1.7.0 - 2017-11-08
+This is a feature and maintenance release.
+
+Feature:
+  * (EZ-111) Add support for RPM triggers. You can now add postinstall triggers
+    that run on either installs or upgrades via the
+    `redhat-postinst-install-triggers` or `redhat-postinst-upgrade-triggers`
+    variables under `:lein-ezbake` in your project.clj. These variables are
+    arrays of hashes in the format [ { :package "package", :scripts ["script 1", "script2"] } ]
+  * (EZ-113) Add support for Debian triggers. This adds support for activate
+    triggers via the `debian-activated-triggers` variable. This variable takes
+    an array of trigger names. This also adds support for interest triggers for
+    either install or upgrade via the `debian-interested-upgrade-triggers` and
+    `debian-interested-install-triggers`. These variables are arrays of hashes
```

Comment:
(nitpick) Have the install variable before the upgrade variable, since that's the order you first mention them and to be consistend with the rpm ones.

also missing the word `variables` at the end of that sentence.